solutions/cf-1316E.mdx

```diff
@@ -14,104 +14,163 @@ author: Sofia Yang
 **Time Complexity:** $\mathcal{O}(N\cdot p\cdot 2^p)$
 
 <LanguageSection>
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+using std::cout;
+using std::endl;
+using std::vector;
+using std::pair;
```

Comment:
I thought we standardized bits/stdc++.h and using namespace std :c

Reply:
oh we did?